src/patterns/gen-ai/aws-aoss-cw-dashboard/README.md

```diff
@@ -0,0 +1,209 @@
+# aws-aoss-cw-dashboard
+
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+
+<!--END STABILITY BANNER-->
+
+
+| **Language**                                                                                   | **Package**                             |
+| :----------------------------------------------------------------------------------------------- | ----------------------------------------- |
+| ![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) TypeScript | `@cdklabs/generative-ai-cdk-constructs` |
+| ![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python             | `cdklabs.generative_ai_cdk_constructs`  |
+
+## Table of contents
+
+- [aws-aoss-cw-dashboard](#aws-aoss-cw-dashboard)
+  - [Table of contents](#table-of-contents)
+  - [Credits](#credits)
+  - [Overview](#overview)
+  - [Initializer](#initializer)
+  - [Pattern Construct Props](#pattern-construct-props)
+  - [Pattern Properties](#pattern-properties)
+  - [Methods](#methods)
+  - [Default properties](#default-properties)
+  - [Cost](#cost)
+  - [Security](#security)
+  - [Supported AWS Regions](#supported-aws-regions)
+  - [Quotas](#quotas)
+
+## Credits
+
+Thanks to @rddefauw for the initial version of this construct.
+
+## Overview
+
+This construct provides an Amazon CloudWatch dashboard to monitor metrics on Amazon OpenSearch Serverless usage. The specific list of metrics created by this construct is available [here](#default-properties). More information about the metrics displayed can be found [here](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring-cloudwatch.html).
+
```

Comment:
can we add the list of metrics which are displayed on the dashboard. Or may be attach a screen shot of the dashboard. Couldn't find the list of metrics in the mentioned link - https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring-cloudwatch.html

Reply:
Yes they are described in the #default-properties section